<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUiEdgeStylePanel</class>
 <widget class="QWidget" name="CUiEdgeStylePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUiEdgeStylePanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="m_colorGridHolder" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_btnLineWeight">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Border Weight</string>
       </property>
       <property name="text">
        <string>Weight</string>
       </property>
       <property name="icon">
        <iconset resource="../gui/gui.qrc">
         <normaloff>:/res/icons/borderweight_16x16.png</normaloff>:/res/icons/borderweight_16x16.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnLineStyle">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Border Styles</string>
       </property>
       <property name="text">
        <string>Pattern</string>
       </property>
       <property name="icon">
        <iconset resource="../gui/gui.qrc">
         <normaloff>:/res/icons/borderstyle_16x16.png</normaloff>:/res/icons/borderstyle_16x16.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../gui/gui.qrc"/>
 </resources>
 <connections/>
</ui>
